### ISSUE TITLE
Add translations

### ISSUE THREAD

**Issue by chrischi1989** [chrischi1989/EternalVoid#3]
No Text

**Comment by JorisVanEijden** [comment]
Too bad it's all in german. I was looking for a laravel game project to work on.
But my german is not good enough to comfortably work on the codebase.

**Comment by chrischi1989** [comment]
Yeah the name of the buildings, techs, ships, resources and the GUI (HTML) itself is currently in german. These names are used in the code but most of it should be understandable without knowing german. I'll try to add translations as soon as I can.

**Comment by JorisVanEijden** [comment]
Would you consider changing the application to english, or are you planning to go multilingual?
Supporting multiple languages in one application can be tricky. Especially with a global audience where an english language alliance member may not understand what you mean when you want to trade 'wasserstoff'.

**Comment by chrischi1989** [comment]
As you know it's not a problem to translate the GUI into other languages when you're using Laravel. Even Klingon would be possible but if you are concerned about users from various countries playing together, I would rather consider to provide multiple game instances. One for german players, one for english players and so on.

Unfortunately I'm currently quite busy at my workplace so I nearly have no time left to continue working on the game :(

**Comment by JorisVanEijden** [comment]
Okay, we'll make it multilingual, starting with english and german.
I'll start work on the transition and the translations.
I do have some time to spend on it so you'll see some pull requests soon.

**Comment by chrischi1989** [comment]
I'm looking forward to it. :+1:

**Comment by JorisVanEijden** [comment]
I've been looking into several ways to provide localization:

1. [stevebauman/translation](https://github.com/stevebauman/translation)
    - pro: Fully automatic, even translations! Strings in source can be the english version.
    - con: Translations are only editable in database
2. [barryvdh/laravel-translation-manager](https://github.com/barryvdh/laravel-translation-manager)
    - pro: Includes web interface to manage translations.
    - con: need to use dot notated string keys
3. [potsky/laravel-localization-helpers](https://github.com/potsky/laravel-localization-helpers)
    - pro: uses standard laravel localization
    - con: need to use dot notated string keys
4. [xinax/laravel-gettext](https://github.com/xinax/laravel-gettext)
    - pro: creates .po files for editing in poedit
    - con: depends on gettext installed on server
5. [plain laravel](http://laravel.com/docs/5.1/localization)
    - pro: no extra dependencies
    - con: need to use dot notated string keys and eveything needs to be managed manually

Do you have a preference?

**Comment by JorisVanEijden** [comment]
[My branch for this issue](https://github.com/JorisVanEijden/EternalVoid/tree/issue/3/localization)
